### DISCUSSION TITLE
LazyView

### DISCUSSION THREAD

**Discussion by fgiacomelli** [CommunityToolkit/Maui#878] [New Feature Discussions]
Hello, what about lazyview to load the user interface giving a feedback to the user without the horrible effect of freezed app?
I'm writing an app with complex view and I think it should be a nice to have, because at this time when I load my views the first rendering requires too much time for a good user experience.
I recently discovered the delayedview of Jean-Marie Alfonsi but if I could find a similar solution in the official community toolkit that I'm already using for other stuff it would be nice.
Can anyone tell me if it will be integrated or if there is currently a workaround for MAUI?

**Comment by VladislavAntonyuk** [comment]
you can track status here: https://github.com/CommunityToolkit/Maui/issues?q=is%3Aissue+is%3Aopen+lazyview

**Reply by fgiacomelli** [reply]
Thank you very much I will follow the proposal (but at this time I can't figure out when it will be ported to maui)

**Reply by pictos** [reply]
@fgiacomelli we also don't when this will be shipped. What I can say is, LazyView is getting higher priority, since I can see lot of members asking for it (:

**Comment by CliffAgius** [comment]
Closed and answered
